### ISSUE TITLE
Visual model for ARH missiles,Home On Jam(HOJ) for ARH missiles and SARH missiles

### ISSUE THREAD

**Issue by F8fate** [BrettRyland/BDArmory#631]
**Is your feature request related to a problem? Please describe.**
Visual model:for ARH missile,Fire a ARH missile without locking any targets on the aircraft radar, the ARH missile's active radar will turn on and lock in any target in the Fov.
![M9{O)2R0Q$Y$ZWP}MD O7 Q](https://github.com/BrettRyland/BDArmory/assets/151370768/8b189a39-8be9-4caa-ac08-bec4c3c9578d)
Home on jam(HOJ): The seeker is completely passive and it just homes in on radiation (normally reflected from the target by the firing platform's RADAR, or if capable, radiation emitted by a jammer), so long as it's within the seeker boresight.If you fire a ARH/SRAH missile in this situation, the missile will track the radiation and there will be no awareness on the enemy's RWR until the missile hits the target.
![)EMY3CRLG7DE3V2@R}X`JFD](https://github.com/BrettRyland/BDArmory/assets/151370768/c5c1b5f9-ad1f-4e94-b326-dfeb86f62cd3)
![472 _2(6 LK$2)`)F{4@SN2](https://github.com/BrettRyland/BDArmory/assets/151370768/8f63ae4b-9bf7-41e2-8743-5e39d53b7f90)

**Describe the solution you'd like**
for visual mode,I'd like to launch an ARH missile without any target being locked by the radar, and its seeker will immediately turn on and lock and track any target within the FOV.
for Home on jam(HOJ),I'd like In the event that the radar does not lock on to any target, the ARH/SRAH missile seeker will locks on to the jammer that is turned on, just like the anti-radiation missile.

**Additional context**
This is the source I quoted
SRAH HOJ:
[Combat Echo Weapon System Evaluation Program. F-4 AIM-7 Aim-9E Sequential Attack Profile in an ECM Environment..pdf](https://github.com/BrettRyland/BDArmory/files/15431282/Combat.Echo.Weapon.System.Evaluation.Program.F-4.AIM-7.Aim-9E.Sequential.Attack.Profile.in.an.ECM.Environment.pdf)
Combat Echo Weapon System Evaluation Program. F-4/AIM-7/Aim-9E Sequential Attack Profile in an ECM Environment :1-3.

Visual Mode,ARH HOJ:
[Distributed Simulation Testing for Weapons System Performance of the FA-18 and AIM-120.pdf](https://github.com/BrettRyland/BDArmory/files/15431322/Distributed.Simulation.Testing.for.Weapons.System.Performance.of.the.FA-18.and.AIM-120.pdf)
Distributed Simulation Testing for Weapons System Performance ofthe F/A-18 and AIM-120 AMRAAM:4-5.

**Comment by BillNyeTheIE** [comment]
Regarding firing ARH missiles maddog, this was already a feature, however a bug prevented it from working properly, requiring you to be fairly close and pointed fairly accurately for the missile to acquire lock. This has now been rectified. For HOJ however, that would require dual-mode seekers, which is a major undertaking. Don't expect that soon.

**Comment by BillNyeTheIE** [comment]
Can you confirm that maddog/visual launches work as you expect them to in the latest update?

**Comment by F8fate** [comment]
Yes,I tried the 1.6.12 version of BDA,the ARH visual model Running quite well,thank you very much!

**Comment by F8fate** [comment]
@BillNyeTheIE hello,In 1.6.12 version.
for ARH,when targetingType = radar,the Visual model is work well.
but when targetingType = inertial,The Visual model doesn't work.
Is this a issue？
